### DISCUSSION TITLE
Assessment of the chances of building the gtk-maui on Linux

### DISCUSSION THREAD

**Discussion by lytico** [lytico/maui#291] [Announcements]
# building
## as of https://github.com/jsuarezruiz/maui-linux, currently branch https://github.com/jsuarezruiz/maui-linux/commit/c005e3aff8cf64c792f54f7b06b1f9e4ee310f78

it works.
 
## as of actual version of https://github.com/dotnet/maui/
last try to get in sync is here: https://github.com/lytico/maui/tree/lytico/merge/main240516
but failed to build

## what is missing
see: https://github.com/users/jsuarezruiz/projects/5

##
@thomiel concrete your question

**Comment by thomiel** [comment]
Using a Docker container, it was surprisingly simple to get it running. So assessment result is 100%.

![image](https://github.com/user-attachments/assets/7342e2cd-b4ab-45f0-a4bf-836bf1f30cf8)

Have a look at the readme of the repo for details: https://github.com/Thomas-Mielke-Software/maui-docker

@lytico Please keep the discussion open, so I can share further findings.
